libafl/src/executors/timeout.rs

```diff
@@ -82,16 +98,70 @@ impl<E> TimeoutExecutor<E> {
     }
 
     #[cfg(windows)]
-    pub fn new(executor: E, exec_tmout: Duration) -> Self {
-        Self { executor }
+    pub unsafe fn new(executor: E, exec_tmout: Duration) -> Self {
+        let milli_sec = exec_tmout.as_millis() as u32;
```

Comment:
Why should `new` be unsafe?

Reply:
no need to be `unsafe`
I forgot to delete it 😅